src/machines/java/com/enderio/machines/common/blockentity/solar/ISolarPanelTier.java

```diff
@@ -0,0 +1,6 @@
+package com.enderio.machines.common.blockentity.solar;
+
+public interface ISolarPanelTier {
+    int getProductionRate();
```

Comment:
Could these return capacitor scalables instead of ints; this way we can add solar power settings to config.